src/nni_manager/training_service/reusable/channels/fileCommandChannel.ts

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) Microsoft Corporation
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+ * BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
```

Comment:
This license section is not consistent with other source files.

Reply:
I copied from other files. Which one is correct reference?

Reply:
Reference nni repo, such as this one: https://github.com/microsoft/nni/blob/master/src/nni_manager/core/commands.ts

Reply:
fixed all incosistent ts files, but some python files are use this format.